docs/ru/sql-reference/functions/arithmetic-functions.md

```diff
@@ -56,7 +56,7 @@ SELECT toTypeName(0), toTypeName(0 + 0), toTypeName(0 + 0 + 0), toTypeName(0 + 0
 ## modulo(a, b), оператор a % b {#modulo}
 
 Вычисляет остаток от деления.
-Если аргументы - числа с плавающей запятой, то они предварительно преобразуются в целые числа, путём отбрасывания дробной части.
+Тип результата - целое число, если оба входа - целые числа. Если один из входов является числом с плавающей точкой, результатом будет число с плавающей точкой.
```

Comment:
Translated with Deepl. Native speakers, please fix that up if the text sounds awkward.